.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    tags: "*"
+
+jobs:
+  push_to_dockerhub:
+    strategy:
+      matrix:
+        image:
+          - {folder: service, dockerhub_name: nia-gp2gp-adaptor}
```

Comment:
Minor: Slightly strange to use a matrix for a single image, but not a bad thing!